screens/settings/NameChangeScreen.js

```diff
@@ -0,0 +1,163 @@
+import { FontAwesome5 } from '@expo/vector-icons';
+import { Updates } from 'expo';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { AsyncStorage, View } from 'react-native';
+import AuthTextField from '../../components/AuthTextField';
+import {
+  BigTitle,
+  ButtonLabel,
+  FilledButtonContainer,
+  Subhead,
+} from '../../components/BaseComponents';
+import Colors from '../../constants/Colors';
+import { getCustomersById, updateCustomers } from '../../lib/airtable/request';
+import { inputFields } from '../../lib/authUtils';
+import {
+  AuthScreenContainer,
+  BackButton,
+  FormContainer,
+} from '../../styled/auth';
+
+export default class NameChangeScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      customer: null,
+      success: false,
+      values: {
+        [inputFields.NAME]: '',
+      },
+      errors: {
+        [inputFields.NAME]: '',
+      },
```

Comment:
Since there is only one type of `inputFields` in this page it is a little unnecessary to specify `inputFields.NAME`/have a dictionaries for values and errors.

Same comment for `PhoneNumberChangeScreen` too!

Reply:
gonna keep this since i copied a lot of code across the files and it might break things if i mess with it